content/en/blog/2024/otel-operator-q-and-a/index.md

```diff
@@ -0,0 +1,345 @@
+
+See the
+[Target Allocator readme](https://github.com/open-telemetry/opentelemetry-operator/tree/main/cmd/otel-allocator#rbac)
+for more on Target Allocator RBAC configuration.
+
+### Q8: Can I override the Target Allocator base image?
+
+Just like you can override the Collector base image in the
+`OpenTelemetryCollector` CR, you can also override the Target Allocator base
+image.
+
+Please keep in mind that
+[it’s usually best to keep the Target Allocator and OTel operator versions the same](https://cloud-native.slack.com/archives/C033BJ8BASU/p1709128862949249?thread_ts=1709081221.484429&cid=C033BJ8BASU),
+to avoid any compatibility issues. If do you choose to override the Target
+Allocator’s base image, you can do so by adding `spec.targetAllocator.image` in
+the `OpenTelemetryCollector` CR. You can also specify the number of replicas by
+adding `spec.targetAllocator.replicas`. This is totally independent of whether
+or not you override the TA image.
+
+Your code would look something like this:
+
+```yaml
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: otelcol
+  namespace: mynamespace
+spec:
+  mode: statefulset
+  targetAllocator:
+    image: <ta_image_name>
+    replicas: <number_of_replicas>
+```
+
+Where:
+
+- `<ta_image_name>` is a valid Target Allocator image from a container
+  repository.
+- `<number_of_replicas>` is the number of pod instances for the underlying
+  Target Allocator
+
+### Q9: If it’s not recommended that you override the Target Allocator base image, then why would you want to?
+
+One use case might be
+[if you need to host a mirror of the Target Allocator image in your own private container registry for security purposes](https://cloud-native.slack.com/archives/C033BJ8BASU/p1713894678225579).
+
+If you do need to reference a Target Allocator image from a private registry,
+you’ll need to use `imagePullSecrets`. To use `imagePullSecrets` with the OTel
+Operator, check out the instructions
+[here](https://github.com/open-telemetry/opentelemetry-operator?tab=readme-ov-file#using-imagepullsecrets).
+Note that you don’t need to create a `serviceAccount` for the Target Allocator,
+since once is already created for you automagically if you don’t create one
+yourself (see
+[Q7](#q7-do-i-need-to-create-a-service-account-to-use-the-target-allocator)).
+
+For more info, check out the
+[Target Allocator API docs](https://github.com/open-telemetry/opentelemetry-operator/blob/main/docs/api.md#opentelemetrycollectorspectargetallocator).
+
+### Q10: Is there a version lag between the OTel Operator auto-instrumentation and auto-instrumentation of supported languages?
+
+If there is a lag, it's minimal, as maintainers try to keep these up to date for
+each release cycle. Keep in mind that there are breaking changes in some
+semantic conventions and the team is trying to avoid breaking users' code. More
+info
+[here](https://cloud-native.slack.com/archives/C033BJ8BASU/p1713894678225579).
```

Comment:
```suggestion
semantic conventions and the team is trying to avoid breaking users' code. For details, see this 
[`#otel-operator` thread](https://cloud-native.slack.com/archives/C033BJ8BASU/p1713894678225579).
```